spec/models/item_spec.rb

```diff
@@ -1,24 +1,24 @@
 require "rails_helper"
 
 RSpec.describe Item, type: :model do
+  let(:category) { Category.create(name: "new cat") }
 
-  let(:valid_attributes) {
-    { title: "food",
-      description: "good",
-      price: 5,
-      status: "active"}
-    }
+  let(:valid_attributes) { { title: "food",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.